### ISSUE TITLE
Make ThreePartDate work with different font sizes

### ISSUE THREAD

**Issue by BryceStevenWilley** [SuffolkLITLab/docassemble-ALToolbox#142]
Our ILAO partners mentioned that with their font size (which should be in their https://github.com/IllinoisLegalAidOnline/docassemble-ILAO repo), the months get cut off in the Three part date.

Ideally, the fields should be responsive enough to fit the longest month length with the current font size. ILAO has expressed interest in submitting a PR, but we might get to it as well.

**Comment by BryceStevenWilley** [comment]
Duplicate of #143
